cmake/Installation.cmake

```diff
@@ -32,9 +32,20 @@ if(APPLE)
   " COMPONENT Runtime
   )
 else()
+  include( GNUInstallDirs )
+  # follow the xdg-desktop specification
+  set(QTOX_DESKTOP_DIR "${CMAKE_INSTALL_FULL_DATAROOTDIR}/applications")
+  set(QTOX_DESKTOP_FILE "io.github.qtox.qTox.desktop")
   install(TARGETS ${PROJECT_NAME} RUNTIME DESTINATION "bin")
-  install(FILES "res/qTox.appdata.xml" DESTINATION "share/metainfo")
-  install(FILES "qtox.desktop" DESTINATION "share/applications")
+  install(FILES "res/io.github.qtox.qTox.appdata.xml" DESTINATION "${CMAKE_INSTALL_FULL_DATAROOTDIR}/metainfo")
+  install(FILES "${QTOX_DESKTOP_FILE}" DESTINATION "${QTOX_DESKTOP_DIR}")
+  
+  # try to set up the menu system
+  find_program(XDG-DESKTOP-MENU_EXECUTABLE xdg-desktop-menu)
```

Comment:
Don't use `xdg-*`, they are pretty bad tools. Just install it normally, use `desktop-file-validate` as part of your build system during `test`. Use `update-desktop-database` as a post-install step.

Reply:
So should I just remove that line? If it's not strictly needed I'd prefer that.

Reply:
It is not needed. `update-desktop-database` is needed for non-packaged installs (i.e. end users installing it directly).

Reply:
ok, I removed that.